### PR TITLE
Fix 1434 - Understand Armory Version in CI Tests

### PR DESCRIPTION
**Pull request by shenshaw26**

### CODE REVIEW

**Comment by shenshaw26** [comment]
I think something else was going on, bu tall these tests pass without the fetch-depth bit.  See:

https://github.com/twosixlabs/armory/runs/6196743409?check_suite_focus=true

I recommend we merge this.

File: .github/workflows/ci_test.yml [hunk 1]
```diff
@@ -120,6 +115,7 @@ jobs:
           pip install -r requirements.txt
           pip install -e .
           python docker/build.py tf2
+          echo "armory version: $(armory --version)"
```
Comment (davidslater) [inline .github/workflows/ci_test.yml:118]: Remove, unless you think it makes sense to add to the other docker tests.

Comment (shenshaw26) [reply .github/workflows/ci_test.yml]: done.

**Review by davidslater — APPROVED** [review]
LGTM. Unclear why that test had failed.

Armory versions are wrong, but they are consistent across tests and tied to the current commit.

**Pull request merged**